server/db/events.py

```diff
@@ -1,8 +1,8 @@
 """PURPOSE: https://fastapi.tiangolo.com/advanced/events/
 
 TODO: https://docs.sqlalchemy.org/en/14/core/connections.html
-    * Investigate whether there are any significant advantages with managing connections ourselves.
-    * Alternative: use `get_db` with Dependency hook for every API request.
+    * Investigate any advantages managing connections ourselves.
+    * Alternative: `get_db` with Dependency hook for every request.
```

Comment:
We still have to do these lol

Reply:
This seems to be a duplicate comment. See my response [here](https://github.com/clairBuoyant/server/pull/40#discussion_r936277512).